cmd/notifier/main.go

```diff
@@ -96,16 +96,13 @@ func main() {
 	}
 
 	// Start moira self state checker
-	selfState := &selfstate.SelfCheckWorker{
-		Logger:   logger,
-		Database: database,
-		Config:   config.Notifier.SelfState.getSettings(),
-		Notifier: sender,
-	}
-	if err := selfState.Start(); err != nil {
-		logger.Fatalf("SelfState failed: %v", err)
+	if config.Notifier.SelfState.getSettings().Enabled {
```

Comment:
It seems like we used to write log when service was disabled. Now we don't write anything in suach case. Maybe we should do it here?

Reply:
Thanks! There was log.Debug, fixed https://github.com/moira-alert/moira/pull/807/commits/dc691e3351d78a2221cf7de75aaf4cf3cf399968